pinot-integration-tests/src/test/java/com/linkedin/pinot/integration/tests/RealtimeClusterIntegrationTest.java

```diff
@@ -109,6 +110,41 @@ public void testGeneratedQueriesWithMultiValues() throws Exception {
     super.testGeneratedQueriesWithMultiValues();
   }
 
+  /**
+   * In realtime consuming segments, the dictionary is not sorted,
+   * and the dictionary based operator should not be used
+   *
+   * Adding explicit queries to test dictionary based functions,
+   * to ensure the right result is computed, wherein dictionary is not read if it is mutable
+   * @throws Exception
+   */
+  @Test
+  public void testDictionaryBasedQueries() throws Exception {
+
+    // Dictionary columns
+    // int
+    testDictionaryBasedFunctions("NASDelay");
```

Comment:
can we not go over the schema and test this for all columns instead of hardcoding the column names